Misc/NEWS.d/next/IDLE/2019-11-23-21-50-57.bpo-38862.KQ9A0m.rst

```diff
@@ -0,0 +1,4 @@
+Format menu Strip Trailing Whitespace adjusts ending newlines of non shell
+windows so that the file is either blank or ends with one newline after
+something other than space, tab, or newline.  The result should be ready to
```

Comment:
I find the comment about committing to the CPython repo is a bit strange, since IDLE is certainly not mainly used for CPython development. I think that this behavior is simply obviously good.

Reply:
After changing the issue spec, this is no longer true.  Removed.